zebra-state/src/service/arbitrary.rs

```diff
@@ -86,3 +91,46 @@ impl Strategy for PreparedChain {
         })
     }
 }
+
+/// Generate a chain to test legacy chain check.  
```

Comment:
Documentation: can you explain what the strategy does, and what each argument does?

For arguments that you're passing to the inner strategy, it's better to say "See `LedgerState::height_strategy` for details." It's less work for you, and we don't have to update multiple docs when the arguments change.

Reply:
Added some docs at https://github.com/ZcashFoundation/zebra/pull/2366/commits/93530423bd92933c32208b7e0e9dba2c204f0a67 and https://github.com/ZcashFoundation/zebra/pull/2366/commits/591eff1d906aa5ce258d569f7cd76e3d63366b2b